erica_app/erica/routes.py

```diff
@@ -55,6 +55,22 @@ def send_est(est: EstData, include_elster_responses: bool = False):
         raise HTTPException(status_code=422, detail=e.generate_error_response(include_elster_responses))
 
 
+@app.post(ERICA_VERSION_URL + '/grundsteuer', status_code=status.HTTP_201_CREATED)
+def send_grundsteuer(grundsteuer_data: GrundsteuerData, include_elster_responses: bool = False):
+    """
+    The Grundsteuer data is validated and then send to ELSTER using ERiC. If it is successful, this should return a 200
+    HTTP response with {'transfer_ticket': str, 'pdf': str}. The pdf is base64 encoded binary data of the pdf
+    If there is any error with the validation, this should return a 400 response. If the validation failed with
+    {‘code’ : int,‘message’: str,‘description’: str, ‘validation_problems’ : [{‘code’: int, ‘message’: str}]}
+    or a 400 repsonse for other client errors and a 500 response for server errors with
+    {‘code’ : int,‘message’: str,‘description’: str}
```

Comment:
I guess you copied that from est. Is it going to be the same for the grundsteuer (transfer_ticket and pdf)? Otherwise maybe keep only the part that you already know and remove the rest because I could imagine that we do not come back to change the documentation and someone gets confused in the future?

Reply:
Also, 

> If it is successful, this should return a 200 HTTP response

does "this" refer to the synchronous return value of the same method or is there more going on here? Because the function annotation specifies 201 on success.

Reply:
>  Is it going to be the same for the grundsteuer (transfer_ticket and pdf)?

Very valid question, I indeed didn't check before. Now, I looked it up and yes, it's the same. PDT and transfer ticket are both generated. So I'd leave that part.

> does "this" refer to the synchronous return value of the same method or is there more going on here? Because the function annotation specifies 201 on success.

Very good question and I actually asked myself that when writing the docu. I settled on the following explanation and @JulianRoesner please correct me if I'm wrong: "a 200 HTTP response" means "anything that start with 2 HTTP response", so including 201. But I think you do have a point Shahrooz and I'd opt for changing that to 201.

Reply:
Thanks for the explanation! The common notation for "a two hundred something response" is 2xx and I recommend using it since it eliminates the ambiguity.

Reply:
Great, thankss, now I remember that I've actually seen this a lot. :D 🤦‍♀️